Dockerfile

```diff
@@ -20,46 +20,25 @@ RUN mkdir -p $ADAPTER_ROOT/
 
 COPY . $ADAPTER_ROOT/
 
+WORKDIR $ADAPTER_ROOT
 # If LOCAL_AETHER_MODELS was used, then patch the go.mod file to load
 # the models from the local source.
 RUN if [ -n "$LOCAL_AETHER_MODELS" ] ; then \
     echo "replace github.com/onosproject/aether-models/models/aether-4.x => ./local-aether-models/aether-4.x" >> $ADAPTER_ROOT/go.mod; \
     echo "replace github.com/onosproject/aether-models/models/aether-2.0.x => ./local-aether-models/aether-2.0.x" >> $ADAPTER_ROOT/go.mod; \
+    echo "replace github.com/onosproject/aether-models/models/aether-2.1.x/v2 v2.1.13 => ./local-aether-models/aether-2.1.x" >> $ADAPTER_ROOT/go.mod; \
+    go mod tidy;\
     fi
 
 RUN cat $ADAPTER_ROOT/go.mod
 
 RUN cd $ADAPTER_ROOT && GO111MODULE=on go build -o /go/bin/sdcore-adapter \
-        -ldflags \
```

Comment:
I don't know if all these should be removed - maybe there is some way to fix it?